Content.Shared/SubFloor/SharedTrayScannerSystem.cs

```diff
@@ -46,6 +59,20 @@ private void SetScannerEnabled(EntityUid uid, bool enabled, TrayScannerComponent
         }
     }
 
+    private void SetWearableScannerEnabled(EntityUid uid, bool enabled, WearableTrayScannerComponent? scanner = null)
+    {
+        if (!Resolve(uid, ref scanner) || scanner.Enabled == enabled)
+            return;
+
+        scanner.Enabled = enabled;
+        Dirty(scanner);
```

Comment:
`Dirty(uid, scanner);`

Reply:
i just copied from the method above and it doesnt do this? why does this one need to specify uid? same with setdata

Reply:
without uid is deprecated

Reply:
aight i guess i'll change this for the code that i copied from too for consistency sake